go/hyperkit.go

```diff
@@ -367,32 +375,49 @@ func (h *HyperKit) buildArgs(cmdline string) {
 	a = append(a, "-m", fmt.Sprintf("%dM", h.Memory))
 
 	a = append(a, "-s", "0:0,hostbridge")
+	a = append(a, "-s", "31,lpc")
+
+	nextSlot := 1
```

Comment:
This change was mostly to make the option I added to the API support the same features as the underlying process and make it easier in the future to support multiple other devices.

My use case is getting Docker for Mac running on Linuxkit and it uses two 9p sockets at the moment. It would be nice to have support in linuxkit for this while testing, but eventually one of the sockets will go away so it's not needed in the longer term.

I don't really have a strong opinion about this either. If you think we don't need it I can remove it from the PR.